CMake/config.cmake

```diff
@@ -4,12 +4,14 @@ install(FILES ${CMAKE_BINARY_DIR}/senseiConfig.h DESTINATION include)
 
 set(SENSEI_BUILD OFF)
 
+set(sensei_CMAKE_INSTALL_CMAKEDIR ${sensei_CMAKE_INSTALL_CMAKEDIR}/sensei)
```

Comment:
Is `sensei_CMAKE_INSTALL_CMAKEDIR` set at this point already (to something like `lib/cmake/`)?

Reply:
Copy-paste error!